src/formElementPlugins/types.ts

```diff
@@ -56,7 +56,7 @@ type ValidationState = {
 interface ApplicationViewProps extends ApplicationViewWrapperProps {
   onUpdate: Function
   onSave: Function
-  value: string // TODO: Change to allow object with any shape
+  value: any
   setValue: (text: string) => void // TO update the value on the ApplicationViewWrapper
```

Comment:
Isn't the `value` not used for theDropdown? Why are we changing here. And if the case is to change, should also change the `setValue` to accept any value too?

Reply:
Yeah same, related to the last comment I think.